src/bundles/debug.js

```diff
@@ -18,71 +22,67 @@ export default {
   },
   doEnableDebug: () => ({ type: ENABLE }),
   doDisableDebug: () => ({ type: DISABLE }),
-  selectIsDebug: state => state.debug,
-  getMiddleware: () => debugMiddleware,
-  init: store => {
-    if (store.selectIsDebug()) {
-      const names = store.meta.chunks[0].bundleNames
-      self.store = store
-      const actionCreators = []
-      for (const key in store) {
-        if (key.indexOf('do') === 0) {
-          actionCreators.push(key)
-        }
-      }
-      actionCreators.sort()
-      const colorTitle = 'color: #1676D2;'
-      const black = 'color: black;'
-      const colorGreen = 'color: #4CAF50;'
-      const colorOrange = 'color: #F57C00;'
-
-      store.logSelectors = self.logSelectors = () => {
-        if (!store.selectAll) return
-        console.log('%cselectors:', colorGreen, store.selectAll())
+  doLogBundles: () => ({ store }) => {
+    const names = store.meta.chunks[0].bundleNames
+    return console.log(
+      '%cinstalled bundles:\n  %c%s',
```

Comment:
I know this isn't new, so feel free to leave it alone, but since you're in here anyway, feel free to look at why the display of the bundle names is so odd in the browser:

![screen shot 2018-08-03 at 7 41 17 am](https://user-images.githubusercontent.com/67925/43648983-c11c98ea-96f0-11e8-9da6-e8aca99ad062.png)